src/plugin/context.rs

```diff
@@ -454,7 +454,7 @@ impl RapierContext {
                         query_pipeline,
                         (&scaled_shape).into(),
                         shape_mass,
-                        collisions.iter().copied(),
+                        collisions.iter(),
```

Comment:
an alternative is to pass `&*collisions`, to "reborrow" is immutably. I'm not sure if that needs a migration guide (it has to be compared with `bevy_rapier 0.26`, not master)